### DISCUSSION TITLE
PiranhaCMS .NET Version Support (STS or LTS)?

### DISCUSSION THREAD

**Discussion by hedronn** [PiranhaCMS/piranha.core#1957] [General] [answered]
Not sure if this question has been asked before, but what is the intention for .NET version support for PiranhaCMS? Is it to ONLY support .NET (LTS) versions therefore skipping the latest .NET 7 for .NET 8 (LTS) later next year?

I assume its LTS as .NET 6 (even number LTS) is currently supported, but just wanted to clarify for future roadmap requirements.

**Comment by tidyui** [comment, marked as answer]
We usually try to support the latest version of .NET available, however we haven't had time to test it against .NET 7 as it was just released last week. It's however unlikely that we will do a force upgrade for `.NET 7` as 6 is the LTS version (just like you wrote).

**EDIT**

We can probably do a multi-framework target for both `.NET6.0` and `.NET7.0`, however at the moment we need to wait for `Pomelo.EntityFrameworkCore.MySql` to release a version that supports 7.0.

**Reply by hedronn** [reply]
My projects have focused on .NET LTS versions, and will skip .NET 7 for .NET 8 release later 2023 as .NET 6 (LTS) remains supported until then. Thanks for the update however its good to know the approach PiranhaCMS is taking. Thank you for the prompt reply as always.
